tests/mocha/utils_test.js

```diff
@@ -23,73 +23,124 @@ suite('Utils', function() {
   });
 
   suite('tokenizeInterpolation', function() {
-    test('Basic', function() {
-      var tokens = Blockly.utils.tokenizeInterpolation('');
-      chai.assert.deepEqual(tokens, [], 'Null interpolation');
-
-      tokens = Blockly.utils.tokenizeInterpolation('Hello');
-      chai.assert.deepEqual(tokens, ['Hello'], 'No interpolation');
-
-      tokens = Blockly.utils.tokenizeInterpolation('Hello%World');
-      chai.assert.deepEqual(tokens, ['Hello%World'], 'Unescaped %.');
+    setup(function() {
+      this.assertInterpolation = function(string, tokens) {
+        chai.assert.deepEqual(
+            Blockly.utils.tokenizeInterpolation(string), tokens);
+      };
+    });
 
-      tokens = Blockly.utils.tokenizeInterpolation('Hello%%World');
-      chai.assert.deepEqual(tokens, ['Hello%World'], 'Escaped %.');
+    suite('Basic', function() {
```

Comment:
I like that these are broken into separate tests now, but I don't think you are gaining much by using the assertInterpolation function. In general, unless the series of assertions are really complex, I think tests are better off being very plain and explicit about their assertions for ease of reading and so that you can tell a test is correct at first glance. And that way you can add individual error messages (though I will say that some of the original error messages of "Interpolations." are comically unhelpful).

Reply:
The individual error messages thing is a good point! I'll have to keep that in mind in the future.